doc/examples/gatk/README.md

```diff
@@ -0,0 +1,156 @@
+
+```sh
+$ cd data/bams/
+$ pachctl start-commit samples master
+$ for f in $(ls mother.*); do pachctl put-file samples 64f1d3456e184efa8f8c9ea7a2994edd mother/$f -f $f; done
+$ pachctl finish-commit samples 64f1d3456e184efa8f8c9ea7a2994edd
+$ cd ../../
+```
+
+You should then be able to see the versioned sample data in Pachyderm:
+
+```sh
+$ pachctl list-file samples master
+NAME                TYPE                SIZE
+mother              dir                 23.79MiB
+$ pachctl list-file samples master mother
+NAME                TYPE                SIZE
+mother/mother.bai   file                9.047KiB
+mother/mother.bam   file                23.79MiB
+```
+
+## Variant calling
+
+To call variants for the input sample, we will run the `HaplotypeCaller` using GATK. Details of the exact GATK command and related scripting are included in the [likelihoods.json](likelihoods.json) pipeline specification. 
+
+To create and run the variant calling pipeline to generate genotype likelihoods:
+
+```sh
+$ pachctl create-pipeline -f likelihoods.json
+```
+
+This will automatically trigger a job:
+
+```sh
+$ pachctl list-job
+ID                                   OUTPUT COMMIT                                STARTED        DURATION   RESTART PROGRESS  DL       UL       STATE
+c61c71d1-6544-48ad-8361-b4ad155ba1a0 likelihoods/992393004c5a45c0a35995cf0179f1cb 43 minutes ago 18 seconds 0       1 + 0 / 1 107.5MiB 4.667MiB success
+```
+
+And you can view the output of the variant calling as follows:
+
+```sh
+$ pachctl list-file likelihoods master
+NAME                TYPE                SIZE
+mother.g.vcf        file                4.667MiB
+mother.g.vcf.idx    file                758B
+```
+
+## Joint genotyping
+
+The last step is to joint call all your GVCF files using the GATK tool GenotypeGVCFs.  Details of this GATK command and related scripting are included in the [joint-call.json](joint-call.json) pipeline specification.
+
+To run the joint genotyping:
+
+```sh
+$ pachctl create-pipeline -f joint-call.json
+```
+
+This will automatically trigger a job and produce our final output:
+
+```sh
+$ pachctl list-job
+ID                                   OUTPUT COMMIT                                STARTED        DURATION   RESTART PROGRESS  DL       UL       STATE
+67135f10-4121-4f29-a30b-1eaf6ffe2194 joint-call/c4ebd6dd0c764a97a8d7f3a71f6bb9ce  38 minutes ago 5 seconds  0       1 + 0 / 1 88.35MiB 113.9KiB success
+c61c71d1-6544-48ad-8361-b4ad155ba1a0 likelihoods/992393004c5a45c0a35995cf0179f1cb 43 minutes ago 18 seconds 0       1 + 0 / 1 107.5MiB 4.667MiB success
+$ pachctl list-file joint-call master
+NAME                TYPE                SIZE
+joint.vcf           file                103.7KiB
+joint.vcf.idx       file                10.21KiB
+```
+
+## Adding more samples
```

Comment:
Broad question: Do we want to include the UI or any screenshots in examples like this? Especially this one since many bioinformatics users are potential enterprise customers AND they often like the UI interface. 

It still makes sense for all the commands to be through the CLI I think, but perhaps for the get/list commands above, you can also add screenshots of the UI in addition? What do you think?

Reply:
Yeah, I added a general view at the end. I think most of the examples should be CLI/client focused. But I definitely agree that adding some plugs for the dashboard would be useful.